homework-submission/todoapp/src/Stores/TodoStore.js

```diff
@@ -0,0 +1,151 @@
+import { observable, action, runInAction } from 'mobx';
+import uuidv4 from 'uuid/v4'
+
+
+class TodosStore {
+  @observable todos = []
+  @observable state = 'loading'
```

Comment:
According to the application logic, perhaps you may have another look to the state status here, I meant `loading` 🤔